ui/src/useFetchAuth.js

```diff
@@ -2,57 +2,57 @@ import { useState, useEffect } from "react";
 import { useAuth0 } from "@auth0/auth0-react";
 
 export default function useFetchAuth(url, method = "GET", body = {}) {
-  const [data, setData] = useState([]);
-  const [error, setError] = useState(null);
-  const [loading, setLoading] = useState(true);
-  const [running, setRunning] = useState(false);
-  const { isAuthenticated, getAccessTokenSilently } = useAuth0();
+	const [data, setData] = useState([]);
+	const [error, setError] = useState(null);
+	const [loading, setLoading] = useState(true);
+	const [running, setRunning] = useState(false);
+	const { isAuthenticated, getAccessTokenSilently } = useAuth0();
 
-  useEffect(() => {
-    async function init() {
-      try {
-        const accessToken = await getAccessTokenSilently();
-        console.log(accessToken);
+	useEffect(() => {
+		async function init() {
+			try {
+				const accessToken = await getAccessTokenSilently();
+				console.log(accessToken);
 
-        const options = {
-          method: method,
-          headers: {
-            "Authorization": `Bearer ${accessToken}`,
-          }
-        };
-        switch (method) {
-          case "POST":
-          case "PUT":
-          case "PATCH":
-            options.body = JSON.stringify(body);
-            options.headers["Content-Type"] = "application/json";
-            break;
-          default:
-            break;
-        }
+				const options = {
+					method: method,
+					headers: {
+						"Authorization": `Bearer ${accessToken}`,
```

Comment:
NIT: you might consider setting your spaces to 2 so that when you reformat a file it doesn't change every line like this for this project. It's not a big deal.